SubProject.cmake

```diff
@@ -225,8 +229,13 @@ if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/.gitsubprojects")
         "else()\n"
         "  file(APPEND .gitsubprojects \"git_subproject(${__subproject})\n\")\n"
         "endif()\n")
+      set(__subproject_paths "${__subproject_paths} ${__subproject_dir}")
```

Comment:
If I remember properly, I think I tried that, but I wanted to de-duplicate the list, so I have to change it into a variable at some point and thus it ended up being the same, just slightly re-arranged.